app/src/main/res/values/faqs.xml

```diff
@@ -9,13 +9,13 @@
-  <string name="faq_answer_8"><![CDATA[<p>To download an Exploration:</p><p>1. From the Home Page, tap on a Topic or Exploration.</p><p>2. From that Topic Page, tap on the <strong>Info</strong> tab.</p><p>3. Tap on <strong>Download Topic</strong>. </p><p>4. Depending on your app settings, you may need Administrator approval or stable Wifi connection to complete your download. If needed, once these requirements are satisfied, the Topic has been downloaded onto the device and can be used offline by all profiles. <p>]]></string>
-  <string name="faq_answer_9"><![CDATA[<p>If you cannot find your question or would like to report a bug, contact us at admin@oppia.org.</p>]]></string>
+  <string name="faq_answer_1"><![CDATA[ <span> If it is your first time creating a profile and not have a PIN:  </span> <br/> <br/> <span> 1. From the Profile Chooser, tap on <strong>Set up Multiple Profiles</strong>.  </span> <br/><br/> <span> 2. Create a PIN and <strong>Save</strong>.  </span> <br/><br/> <span> 3. Fill in all fields for the profile.  </span> <ul><li dir="rtl"> (Optional) Upload a photo. </li><li dir="rtl"> Enter a name. </li><li dir="rtl"> (Optional) Assign a 3-digit PIN. </li> </ul>  <span> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser! <br/> <br/> If you have created a profile before and have a PIN:  </span> <br/><br/> <span> 1. From the Profile Chooser, tap on <strong>Add Profile</strong>.  </span> <br/> <br/> <span> 2. Enter your PIN and tap <strong>Submit</strong>.  </span> <br/><br/> <span> 3. Fill in all fields for the profile.  </span> <ul><li dir="rtl"> (Optional) Upload a photo. </li><li dir="rtl"> Enter a name. </li><li dir="rtl"> (Optional) Assign a 3-digit PIN. </li> </ul>  <span> 4. Tap <strong>Create</strong>. This profile is added to your Profile Chooser!   Note: Only the <u>Administrator</u> is able to manage profiles. </span>]]></string>
+  <string name="faq_answer_2"><![CDATA[ <span> Once a profile is deleted: </span><ul><li dir="rtl"> The profile cannot be recovered. </li><li dir="rtl"> Profile information such as name, photos, and progress will be permanently deleted. </li> </ul> <span>To delete a profile (excluding the <u>Administrator\'s</u>): </span> <br/><br/> <span>1. From the Administrator\'s Home Page, tap on the menu button on the top left. </span> <br/><br/> <span>2. Tap on <strong>Administrator Controls</strong>. </span> <br/><br/> <span>3. Tap on <strong>Edit Profiles</strong>. </span> <br/><br/> <span>4. Tap on the Profile you would like to delete. </span> <br/><br/> <span>5. At the bottom of the screen, tap <strong>Profile Deletion</strong>. </span> <br/><br/> <span>6. Tap <strong>Delete</strong> to confirm deletion. </span><br/> <span>Note: Only the <u>Administrator</u> is able to manage profiles. </span>]]></string>
```

Comment:
Might it be possible for us to automatically add the rtl directionality via HtmlParser? One concern I have with this is it may be lost across translations once these strings go through Translatewiki.

Reply:
Do you mean like as below
```
htmlContent = htmlContent.replace("<li>", "<$CUSTOM_BULLET_LIST_TAG dir=\"rtl\">")
          .replace("</li>", "</$CUSTOM_BULLET_LIST_TAG>")
```
This crashes with runtime exception.

Reply:
What's the exception? Without that context it's hard to know whether the solution can be made to work.

Reply:
`java.lang.RuntimeException: PARAGRAPH span must end at paragraph boundary`

Reply:
@veena14cs when providing an exception, you should always provide the full stack trace & error since the error alone doesn't provide the full context.

Also, you should provide context on why this exception can't be worked around. The replace solution is still ideal, and we should try to make it work unless it's clear that & why it won't work (simply running into an exception doesn't tell us this--that could indicate something else is being set up incorrectly).